.github/workflows/code_review.yml

```diff
@@ -12,9 +12,10 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: ./
```

Comment:
Using a relative path for the action might lead to issues if the action is not properly defined in the repository. Consider specifying a version or a specific commit hash instead.